docs/untyped-blame.md

```diff
@@ -0,0 +1,219 @@
+# Blaming usages of untyped to definitions
+
+It's possible to build Sorbet in a way that it can "blame" usages of untyped
+back to a definition (like a method or instance variable) which introduced the
+usage of untyped in the first place. With this information, it becomes easier to
+evaluate the cost vs benefit of improving the type annotations on any given
+method.
+
+It requires building Sorbet with a special build configuration, because simply
+adding the capability to blame untyped usages to definitions (regardless of
+whether the user actually cares to use that capability) causes Sorbet to use
+substantially more memory for certain internal data structures.
+
+Once built, it works something like this. Given an input file like this:
+
+```ruby
+# typed: true
+
+def method_without_a_sig_1 = 0
+def method_without_a_sig_2 = 0
```

Comment:
[non blocking nit] You could put this method inside some class, to show how the `owner` field will change based on that.